docker-for-windows/index.md

```diff
@@ -468,23 +340,25 @@ If `Experimental` is `true`, then Docker is running in experimental mode, as
 shown here. (If `false`, Experimental mode is off.)
 
 ```bash
-PS C:\Users\Vicky> docker version
```

Comment:
That's a lot of cruft to maintain between releases.  I would rather recommend
```
docker info -f '{{.ExperimentalBuild}}'
```
(or maybe single quotes is really PowerShell does not support single quotes).